In [176]:
from pandas import DataFrame, read_csv
import pandas as pd
import matplotlib.pyplot as plt
import numpy as np

# Survival of Passengers of the Titanic

In [177]:
Location = r'C:\Users\jwhit\jupyter\titanic.csv'
titanic = pd.read_csv(Location)
titanic

Unnamed: 0                                              Name PClass  \
0              1                      Allen, Miss Elisabeth Walton    1st   
1              2                       Allison, Miss Helen Loraine    1st   
2              3               Allison, Mr Hudson Joshua Creighton    1st   
3              4     Allison, Mrs Hudson JC (Bessie Waldo Daniels)    1st   
4              5                     Allison, Master Hudson Trevor    1st   
5              6                                Anderson, Mr Harry    1st   
6              7                  Andrews, Miss Kornelia Theodosia    1st   
7              8                            Andrews, Mr Thomas, jr    1st   
8              9      Appleton, Mrs Edward Dale (Charlotte Lamson)    1st   
9             10                            Artagaveytia, Mr Ramon    1st   
10            11                         Astor, Colonel John Jacob    1st   
11            12  Astor, Mrs John Jacob (Madeleine Talmadge Force)    1st   
12            13                      Aubert, Mrs Leontine Pauline    1st   
13            14                          Barkworth, Mr Algernon H    1st   
14            15                                Baumann, Mr John D    1st   
15            16    Baxter, Mrs James (Helene DeLaudeniere Chaput)    1st   
16            17                           Baxter, Mr Quigg Edmond    1st   
17            18                               Beattie, Mr Thomson    1st   
18            19                      Beckwith, Mr Richard Leonard    1st   
19            20   Beckwith, Mrs Richard Leonard (Sallie Monypeny)    1st   
20            21                              Behr, Mr Karl Howell    1st   
21            22                                Birnbaum, Mr Jakob    1st   
22            23                            Bishop, Mr Dickinson H    1st   
23            24            Bishop, Mrs Dickinson H (Helen Walton)    1st   
24            25           Bjornstrm-Steffansson, Mr Mauritz Hakan    1st   
25            26                       Blackwell, Mr Stephen Weart    1st   
26            27                                   Blank, Mr Henry    1st   
27            28                            Bonnell, Miss Caroline    1st   
28            29                           Bonnell, Miss Elizabeth    1st   
29            30                           Borebank, Mr John James    1st   
...          ...                                               ...    ...   
1283        1284               Vestrom, Miss Hulda Amanda Adolfina    3rd   
1284        1285                                    Vonk, Mr Jenko    3rd   
1285        1286                                Ware, Mr Frederick    3rd   
1286        1287                        Warren, Mr Charles William    3rd   
1287        1288                                  Wazli, Mr Yousif    3rd   
1288        1289                                  Webber, Mr James    3rd   
1289        1290                     Wennerstrom, Mr August Edvard    3rd   
1290        1291                                Wenzel, Mr Linhart    3rd   
1291        1292                        Widegren, Mr Charles Peter    3rd   
1292        1293                          Wiklund, Mr Jacob Alfred    3rd   
1293        1294                                 Wilkes, Mrs Ellen    3rd   
1294        1295                                  Willer, Mr Aaron    3rd   
1295        1296                                 Willey, Mr Edward    3rd   
1296        1297                          Williams, Mr Howard Hugh    3rd   
1297        1298                               Williams, Mr Leslie    3rd   
1298        1299                                Windelov, Mr Einar    3rd   
1299        1300                                   Wirz, Mr Albert    3rd   
1300        1301                             Wiseman, Mr Phillippe    3rd   
1301        1302                           Wittevrongel, Mr Camiel    3rd   
1302        1303                                 Yalsevac, Mr Ivan    3rd   
13

## Cleaning up data to solve querys

### Survivers Male Vs. Female

In [178]:
titanic_clean = titanic[['Sex', 'Survived']]
titanic_clean

Sex  Survived
0     female         1
1     female         0
2       male         0
3     female         0
4       male         1
5       male         1
6     female         1
7       male         0
8     female         1
9       male         0
10      male         0
11    female         1
12    female         1
13      male         1
14      male         0
15    female         1
16      male         0
17      male         0
18      male         1
19    female         1
20      male         1
21      male         0
22      male         1
23    female         1
24      male         1
25      male         0
26      male         1
27    female         1
28    female         1
29      male         0
...      ...       ...
1283  female         0
1284    male         0
1285    male         0
1286    male         0
1287    male         0
1288    male         0
1289    male         1
1290    male         0
1291    male         0
1292    male         0
1293  female         1
1294    male         0
1295    male         0
1296    male         0
1297    male         0
1298    male         0
1299    male         0
1300    male         0
1301    male         0
1302    male         1
1303    male         0
1304  female         1
1305    male         0
1306  female         0
1307  female         0
1308    male         0
1309    male         0
1310    male         0
1311    male         0
1312    male         0

[1313 rows x 2 columns]

In [179]:
survived_totals = titanic_clean.groupby(['Sex','Survived'])
survived_totals.size()

Sex     Survived
female  0           154
        1           308
male    0           709
        1           142
dtype: int64

#### By grouping the data by sex we can see above that 308 Females survived Vs. 142 Males

## What class had the most survivers

In [180]:
titanic_class = titanic[['PClass', 'Survived']]
titanic_class

PClass  Survived
0       1st         1
1       1st         0
2       1st         0
3       1st         0
4       1st         1
5       1st         1
6       1st         1
7       1st         0
8       1st         1
9       1st         0
10      1st         0
11      1st         1
12      1st         1
13      1st         1
14      1st         0
15      1st         1
16      1st         0
17      1st         0
18      1st         1
19      1st         1
20      1st         1
21      1st         0
22      1st         1
23      1st         1
24      1st         1
25      1st         0
26      1st         1
27      1st         1
28      1st         1
29      1st         0
...     ...       ...
1283    3rd         0
1284    3rd         0
1285    3rd         0
1286    3rd         0
1287    3rd         0
1288    3rd         0
1289    3rd         1
1290    3rd         0
1291    3rd         0
1292    3rd         0
1293    3rd         1
1294    3rd         0
1295    3rd         0
1296    3rd         0
1297    3rd         0
1298    3rd         0
1299    3rd         0
1300    3rd         0
1301    3rd         0
1302    3rd         1
1303    3rd         0
1304    3rd         1
1305    3rd         0
1306    3rd         0
1307    3rd         0
1308    3rd         0
1309    3rd         0
1310    3rd         0
1311    3rd         0
1312    3rd         0

[1313 rows x 2 columns]

In [181]:
valuelist = [1]
class_survivors = titanic_class[titanic_class.Survived.isin(valuelist)]
class_survivors

PClass  Survived
0       1st         1
4       1st         1
5       1st         1
6       1st         1
8       1st         1
11      1st         1
12      1st         1
13      1st         1
15      1st         1
18      1st         1
19      1st         1
20      1st         1
22      1st         1
23      1st         1
24      1st         1
26      1st         1
27      1st         1
28      1st         1
30      1st         1
31      1st         1
32      1st         1
36      1st         1
37      1st         1
38      1st         1
40      1st         1
41      1st         1
42      1st         1
43      1st         1
47      1st         1
48      1st         1
...     ...       ...
1098    3rd         1
1122    3rd         1
1125    3rd         1
1136    3rd         1
1151    3rd         1
1161    3rd         1
1164    3rd         1
1167    3rd         1
1182    3rd         1
1187    3rd         1
1199    3rd         1
1201    3rd         1
1207    3rd         1
1219    3rd         1
1222    3rd         1
1226    3rd         1
1231    3rd         1
1235    3rd         1
1239    3rd         1
1242    3rd         1
1244    3rd         1
1251    3rd         1
1258    3rd         1
1263    3rd         1
1264    3rd         1
1279    3rd         1
1289    3rd         1
1293    3rd         1
1302    3rd         1
1304    3rd         1

[450 rows x 2 columns]

In [182]:
class_survivors_totals = class_survivors.groupby(['PClass','Survived'])
class_survivors_totals.size()

PClass  Survived
1st     1           193
2nd     1           119
3rd     1           138
dtype: int64

#### By removing any survivor data that did not equal 1 and grouping by PClass, we see that 1st class had the most survivors

# Earthquakes

In [183]:
Location = r'C:\Users\jwhit\jupyter\quakes.csv'
quakes = pd.read_csv(Location)
quakes

Unnamed: 0    lat    long  depth  mag  stations
0             1 -20.42  181.62    562  4.8        41
1             2 -20.62  181.03    650  4.2        15
2             3 -26.00  184.10     42  5.4        43
3             4 -17.97  181.66    626  4.1        19
4             5 -20.42  181.96    649  4.0        11
5             6 -19.68  184.31    195  4.0        12
6             7 -11.70  166.10     82  4.8        43
7             8 -28.11  181.93    194  4.4        15
8             9 -28.74  181.74    211  4.7        35
9            10 -17.47  179.59    622  4.3        19
10           11 -21.44  180.69    583  4.4        13
11           12 -12.26  167.00    249  4.6        16
12           13 -18.54  182.11    554  4.4        19
13           14 -21.00  181.66    600  4.4        10
14           15 -20.70  169.92    139  6.1        94
15           16 -15.94  184.95    306  4.3        11
16           17 -13.64  165.96     50  6.0        83
17           18 -17.83  181.50    590  4.5        21
18           19 -23.50  179.78    570  4.4        13
19           20 -22.63  180.31    598  4.4        18
20           21 -20.84  181.16    576  4.5        17
21           22 -10.98  166.32    211  4.2        12
22           23 -23.30  180.16    512  4.4        18
23           24 -30.20  182.00    125  4.7        22
24           25 -19.66  180.28    431  5.4        57
25           26 -17.94  181.49    537  4.0        15
26           27 -14.72  167.51    155  4.6        18
27           28 -16.46  180.79    498  5.2        79
28           29 -20.97  181.47    582  4.5        25
29           30 -19.84  182.37    328  4.4        17
..          ...    ...     ...    ...  ...       ...
970         971 -25.79  182.38    172  4.4        14
971         972 -23.75  184.50     54  5.2        74
972         973 -24.10  184.50     68  4.7        23
973         974 -18.56  169.05    217  4.9        35
974         975 -23.30  184.68    102  4.9        27
975         976 -17.03  185.74    178  4.2        32
976         977 -20.77  183.71    251  4.4        47
977         978 -28.10  183.50     42  4.4        17
978         979 -18.83  182.26    575  4.3        11
979         980 -23.00  170.70     43  4.9        20
980         981 -20.82  181.67    577  5.0        67
981         982 -22.95  170.56     42  4.7        21
982         983 -28.22  183.60     75  4.9        49
983         984 -27.99  183.50     71  4.3        22
984         985 -15.54  187.15     60  4.5        17
985         986 -12.37  166.93    291  4.2        16
986         987 -22.33  171.66    125  5.2        51
987         988 -22.70  170.30     69  4.8        27
988         989 -17.86  181.30    614  4.0        12
989         990 -16.00  184.53    108  4.7        33
990         991 -20.73  181.42    575  4.3        18
991         992 -15.45  181.42    409  4.3        27
992         993 -20.05  183.86    243  4.9        65
993         994 -17.95  181.37    642  4.0        17
994         995 -17.70  188.10     45  4.2        10
995         996 -25.93  179.54    470  4.4        22
996         997 -12.28  167.06    248  4.7        35
997         998 -20.13  184.20    244  4.5        34
998         999 -17.40  187.80     40  4.5        14
999        1000 -21.59  170.56    165  6.0       119

[1000 rows x 6 columns]

## Cleaning up and organizing the data

In [184]:
del quakes['Unnamed: 0']

In [185]:
quakes.sort_values(['stations'], inplace=True, ascending=True)
quakes

lat    long  depth  mag  stations
174 -15.03  182.29    399  4.1        10
703 -25.00  180.00    488  4.5        10
692 -20.61  182.44    518  4.2        10
689 -17.60  181.50    548  4.1        10
762 -17.78  185.33    223  4.1        10
769 -20.70  186.30     80  4.0        10
775 -21.77  181.00    618  4.1        10
34  -16.30  186.00     48  4.5        10
777 -21.05  180.90    616  4.3        10
481 -14.85  184.87    294  4.1        10
437 -20.30  182.30    476  4.5        10
262 -19.06  169.01    158  4.4        10
430 -18.40  183.40    343  4.1        10
283 -17.70  185.00    383  4.0        10
326 -21.04  181.20    483  4.2        10
145 -20.10  184.40    186  4.2        10
33  -23.55  180.80    349  4.0        10
349 -27.21  182.43     55  4.6        10
994 -17.70  188.10     45  4.2        10
13  -21.00  181.66    600  4.4        10
450 -26.46  182.50    184  4.3        11
186 -22.30  181.90    309  4.3        11
76  -19.21  184.70    197  4.1        11
706 -23.73  179.98    524  4.6        11
978 -18.83  182.26    575  4.3        11
4   -20.42  181.96    649  4.0        11
57  -22.06  180.60    584  4.0        11
487 -17.61  181.20    537  4.1        11
697 -15.43  185.19    249  4.0        11
813 -23.73  183.00    118  4.3        11
..     ...     ...    ...  ...       ...
600 -27.98  181.96     53  5.2        89
175 -32.22  180.20    216  5.7        90
69  -15.46  187.81     40  5.5        91
701 -17.64  177.01    545  5.2        91
447 -20.85  181.59    499  5.1        91
511 -20.90  184.28     58  5.5        92
399 -13.45  170.30    641  5.3        93
652 -11.40  166.07     93  5.6        94
14  -20.70  169.92    139  6.1        94
379 -15.36  186.71    130  5.5        95
166 -26.00  182.12    205  5.6        98
604 -19.89  174.46    546  5.7        99
461 -18.00  180.62    636  5.0       100
274 -22.13  180.38    577  5.7       104
448 -21.11  181.50    538  5.5       104
662 -18.14  180.87    624  5.5       105
371 -19.89  184.08    219  5.4       105
150 -23.34  184.50     56  5.7       106
711 -15.93  167.91    183  5.6       109
935 -19.33  186.16     44  5.4       110
398 -17.84  181.30    535  5.7       112
919 -17.85  181.44    589  5.6       115
557 -22.91  183.95     64  5.9       118
999 -21.59  170.56    165  6.0       119
752 -21.08  180.85    627  5.9       119
934 -20.25  184.75    107  5.6       121
151 -15.56  167.62    127  6.4       122
375 -15.33  186.75     48  5.7       123
635 -18.82  182.21    417  5.6       129
869 -12.23  167.02    242  6.0       132

[1000 rows x 5 columns]

In [186]:
quakes['stations'].value_counts()[:10]

14    39
17    38
21    37
23    35
22    35
16    35
15    34
18    33
27    33
19    29
Name: stations, dtype: int64

#### From the query above we can the top 10 Stations the reported the most earthquakes

In [187]:
quakes.sort_values(['mag'], inplace=True, ascending=False)
quakes

lat    long  depth  mag  stations
151 -15.56  167.62    127  6.4       122
14  -20.70  169.92    139  6.1        94
869 -12.23  167.02    242  6.0       132
16  -13.64  165.96     50  6.0        83
999 -21.59  170.56    165  6.0       119
752 -21.08  180.85    627  5.9       119
557 -22.91  183.95     64  5.9       118
398 -17.84  181.30    535  5.7       112
175 -32.22  180.20    216  5.7        90
868 -21.14  174.21     40  5.7        78
274 -22.13  180.38    577  5.7       104
150 -23.34  184.50     56  5.7       106
604 -19.89  174.46    546  5.7        99
108 -22.55  185.90     42  5.7        76
375 -15.33  186.75     48  5.7       123
296 -24.57  178.40    562  5.6        80
652 -11.40  166.07     93  5.6        94
934 -20.25  184.75    107  5.6       121
166 -26.00  182.12    205  5.6        98
569 -34.68  179.82     75  5.6        79
711 -15.93  167.91    183  5.6       109
648 -37.03  177.52    153  5.6        87
635 -18.82  182.21    417  5.6       129
919 -17.85  181.44    589  5.6       115
530 -15.77  167.01     64  5.5        73
524 -32.45  181.15     41  5.5        81
947 -27.89  182.92     87  5.5        67
357 -17.10  182.68    403  5.5        82
69  -15.46  187.81     40  5.5        91
511 -20.90  184.28     58  5.5        92
..     ...     ...    ...  ...       ...
855 -18.50  185.40    243  4.0        11
721 -17.91  181.48    555  4.0        17
815 -22.12  180.49    532  4.0        14
732 -30.30  180.80    275  4.0        14
388 -10.72  165.99    195  4.0        14
860 -21.03  180.78    638  4.0        14
918 -25.06  182.80    133  4.0        14
726 -17.10  182.80    390  4.0        14
874 -17.80  181.20    530  4.0        15
51  -19.26  184.42    223  4.0        15
482 -22.70  183.30    180  4.0        13
988 -17.86  181.30    614  4.0        12
779 -17.90  181.50    589  4.0        12
25  -17.94  181.49    537  4.0        15
5   -19.68  184.31    195  4.0        12
112 -19.06  182.45    477  4.0        16
749 -25.60  180.30    440  4.0        12
771 -16.40  182.73    391  4.0        16
201 -17.70  182.20    445  4.0        12
825 -21.62  182.40    350  4.0        12
833 -19.70  182.44    397  4.0        12
636 -19.51  183.97    280  4.0        16
235 -23.54  179.93    574  4.0        12
297 -17.94  181.51    601  4.0        16
298 -30.64  181.20    175  4.0        16
793 -28.00  182.00    199  4.0        16
432 -18.55  182.23    563  4.0        17
597 -17.02  182.93    406  4.0        17
993 -17.95  181.37    642  4.0        17
899 -17.82  181.27    538  4.0        33

[1000 rows x 5 columns]

#### By sorting the dataframe by the mag column we can see which stations reported the largest and smallest earthquakes 

# Polar Bear Video Data

In [188]:
Location = r'C:\Users\jwhit\jupyter\polarbear.csv'
pbear = pd.read_csv(Location)
pbear

Bear Datetime_behavior_starts Datetime_behavior_ends  \
0         1        4/7/2014 16:00:22      4/7/2014 16:01:12   
1         1        4/7/2014 17:49:07      4/7/2014 17:49:57   
2         1        4/7/2014 17:50:53      4/7/2014 17:51:31   
3         1        4/7/2014 17:51:31      4/7/2014 17:51:53   
4         1        4/7/2014 17:52:53      4/7/2014 17:53:53   
5         1        4/7/2014 17:54:53      4/7/2014 17:55:53   
6         1        4/7/2014 17:56:53      4/7/2014 17:57:53   
7         1        4/7/2014 17:58:53      4/7/2014 17:59:53   
8         1        4/7/2014 18:00:53      4/7/2014 18:01:53   
9         1        4/7/2014 18:02:53      4/7/2014 18:03:53   
10        1        4/7/2014 18:04:53      4/7/2014 18:05:48   
11        1        4/7/2014 18:05:48      4/7/2014 18:05:53   
12        1        4/7/2014 18:06:53      4/7/2014 18:07:34   
13        1        4/7/2014 18:07:34      4/7/2014 18:07:37   
14        1        4/7/2014 18:07:37      4/7/2014 18:07:53   
15        1        4/7/2014 18:08:53      4/7/2014 18:09:53   
16        1        4/7/2014 18:10:53      4/7/2014 18:11:53   
17        1        4/7/2014 18:12:53      4/7/2014 18:13:53   
18        1        4/7/2014 18:14:53      4/7/2014 18:15:53   
19        1        4/7/2014 18:16:53      4/7/2014 18:17:53   
20        1        4/7/2014 18:18:53      4/7/2014 18:19:53   
21        1        4/7/2014 18:20:53      4/7/2014 18:20:59   
22        1        4/7/2014 18:20:59      4/7/2014 18:21:00   
23        1        4/7/2014 18:21:00      4/7/2014 18:21:53   
24        1        4/7/2014 18:22:53      4/7/2014 18:23:53   
25        1        4/7/2014 18:24:53      4/7/2014 18:25:53   
26        1        4/7/2014 18:26:53      4/7/2014 18:27:53   
27        1        4/7/2014 18:28:53      4/7/2014 18:29:53   
28        1        4/7/2014 18:30:53      4/7/2014 18:31:53   
29        1        4/7/2014 18:32:53      4/7/2014 18:33:53   
...     ...                      ...                    ...   
41968     9        4/21/2016 3:30:03      4/21/2016 3:31:03   
41969     9        4/21/2016 3:32:03      4/21/2016 3:33:03   
41970     9        4/21/2016 3:34:03      4/21/2016 3:34:29   
41971     9        4/21/2016 3:34:29      4/21/2016 3:34:30   
41972     9        4/21/2016 3:34:30      4/21/2016 3:34:32   
41973     9        4/21/2016 3:34:32      4/21/2016 3:34:33   
41974     9        4/21/2016 3:34:33      4/21/2016 3:34:53   
41975     9        4/21/2016 3:34:53      4/21/2016 3:35:03   
41976     9        4/21/2016 3:36:03      4/21/2016 3:37:03   
41977     9        4/21/2016 3:38:03      4/21/2016 3:39:03   
41978     9        4/21/2016 3:40:03      4/21/2016 3:40:09   
41979     9        4/21/2016 3:40:09      4/21/2016 3:40:12   
41980     9        4/21/2016 3:40:12      4/21/2016 3:40:18   
41981     9        4/21/2016 3:40:18      4/21/2016 3:40:39   
41982     9        4/21/2016 3:40:39      4/21/2016 3:40:57   
41983     9        4/21/2016 3:40:57      4/21/2016 3:41:03   
41984     9        4/21/2016 3:42:03      4/21/2016 3:42:08   
41985     9        4/21/2016 3:42:08      4/21/2016 3:42:10   
41986     9        4/21/2016 3:42:10      4/21/2016 3:43:03   
41987     9        4/21/2016 3:44:03      4/21/2016 3:45:03   
41988     9        4/21/2016 3:46:03      4/21/2016 3:47:03   
41989     9        4/21/2016 3:48:03      4/21/2016 3:48:09   
41990     9        4/21/2016 3:48:09      4/21/2016 3:48:10   
41991     9        4/21/2016 3:48:10      4/21/2016 3:48:12   
41992     9        4/21/2016 3:48:12      4/21/2016 3:48:13   
41993     9        4/21/2016 3:48:13      4/21/2016 3:48:19   
41994     9        4/21/2016 3:48:19      4/21/2016 3:48:27   
41995     9        4/21/2016 3:48:27      4/21/2016 3:48:28   
41996     9        4/21/2016 3:48:28      4/21/2016 3:48:29   
41997     9        4/21/2016 3:48:29      4/21/2016 3:48:53   

                  Behavior                                      Comments  
0                   layin

### Cleaning up the data

In [189]:
behavior = pbear[['Bear', 'Behavior']]
behavior

Bear             Behavior
0         1               laying
1         1      rolling in snow
2         1      rolling in snow
3         1              walking
4         1  sitting or standing
5         1  sitting or standing
6         1  sitting or standing
7         1  sitting or standing
8         1  sitting or standing
9         1  sitting or standing
10        1  sitting or standing
11        1              walking
12        1              walking
13        1      rolling in snow
14        1  sitting or standing
15        1              walking
16        1  sitting or standing
17        1  sitting or standing
18        1  sitting or standing
19        1  sitting or standing
20        1  sitting or standing
21        1              walking
22        1      rolling in snow
23        1              walking
24        1                mixed
25        1      rolling in snow
26        1      rolling in snow
27        1          interacting
28        1      rolling in snow
29        1      rolling in snow
...     ...                  ...
41968     9               laying
41969     9  sitting or standing
41970     9               laying
41971     9            adjusting
41972     9           head shake
41973     9            adjusting
41974     9              unknown
41975     9               laying
41976     9              unknown
41977     9              unknown
41978     9               laying
41979     9            adjusting
41980     9               laying
41981     9            adjusting
41982     9      rolling in snow
41983     9            adjusting
41984     9      rolling in snow
41985     9            adjusting
41986     9               laying
41987     9               laying
41988     9               laying
41989     9  sitting or standing
41990     9            adjusting
41991     9           head shake
41992     9            adjusting
41993     9  sitting or standing
41994     9          interacting
41995     9           head shake
41996     9            adjusting
41997     9  sitting or standing

[41998 rows x 2 columns]

In [190]:
behavior['Bear'].value_counts()

5    8051
8    7561
9    6524
7    4987
6    4145
2    3647
4    3335
1    2558
3    1190
Name: Bear, dtype: int64

#### From the query above we can see how many times data was recorded for each bear

In [191]:
valuelist = ['walking']
pbear_walking = behavior[behavior.Behavior.isin(valuelist)]
pbear_walking_count = pbear_walking.groupby(['Bear', 'Behavior'])
pbear_walking_count.size()

Bear  Behavior
1     walking      172
2     walking      130
3     walking      343
4     walking      758
5     walking     1084
6     walking      665
7     walking      480
8     walking     1501
9     walking      773
dtype: int64

#### From the query above we can see that Bear 8 was recorded walking the most and Bear 2 was recorded walking the least

In [192]:
valuelist = ['4/21/2016 3:40:12']
pbear_time = pbear[pbear.Datetime_behavior_starts.isin(valuelist)]
pbear_time

Bear Datetime_behavior_starts Datetime_behavior_ends Behavior Comments
41980     9        4/21/2016 3:40:12      4/21/2016 3:40:18   laying  resting

#### From the query above we can find out what a bear was doing at a specfic time